site/content/articles/matchbox-overview.adoc

```diff
@@ -0,0 +1,270 @@
+---
+title: "Matchbox Overview"
+date: 2020-01-13T16:50:05-05:00
+authors: [jlee@redhat.com]
+categories:
+  -
+tags:
+  - matchbox  
+draft: true
+---
+
+= Coreos Matchbox overview
```

Comment:
Hey @Jooho I think we need to work on a couple things up front in order to establish a purpose and fit for this article. The first question I would like you to answer, both in the title and the intro is "Who is for, and when should someone use it?" We want each article to present an opinion and make suggestions about what problem we're solving, and why someone would have that problem in the first place. That's what differentiates uncontained.io from, say, the OpenShift blog, or a KB article.